### PR TITLE
refactor(shared-data): remove outdated id fields from JSON v6 fixtures

### PR DESCRIPTION
**Pull request by aaron-kulkarni**
replace all instances of Command `id` with `key` in the v6 protocol fixtures

close EXEC-221

# Overview

# Test Plan

# Changelog

Just changed all of v6 fixtures to have a "key" field in Command rather than an "id" field
Updated `shared-data/protocol/fixtures/types/schemaV6/command/index.ts` to properly recognize the key field.

# Review requests

This ticket is like 2 years old so I don't even know if this matters anymore but here it is.

# Risk assessment

Low

### CODE REVIEW

File: shared-data/protocol/types/schemaV6/command/index.ts [hunk 1]
```diff
@@ -21,7 +21,6 @@ export * from './timing'
 export type CommandStatus = 'queued' | 'running' | 'succeeded' | 'failed'
 export interface CommonCommandRunTimeInfo {
   key?: string
-  id: string
```
Comment (sfoster1) [inline shared-data/protocol/types/schemaV6/command/index.ts]: This actually shouldn't be removed - this typescript type is for run time commands, the outputs of the protocol engine. By that time, an ID has been generated.

However, the protocols are from _before_ the engine executes and definitely shouldn't have IDs. So we should remove this specific line.

**Review by sfoster1 — DISMISSED** [review]
Let's change that one line but otherwise, thanks!

**Review by sfoster1 — APPROVED** [review]
Nice, thank you!

**Review by SyntaxColoring — APPROVED** [review]
🔑 

Thank you thank you. Since that ticket was written, we've also added v7 and v8 fixtures, some of which look to have the same problem (probably copy-pasted). We should fix those too, but you don't have to do that all in a single PR if you don't want.

**Pull request merged**